### PR TITLE
Fix report

### PR DESCRIPTION
**Pull request by iaindillingham**
This PR contains several small fixes to the report, which I noticed when reviewing it on <https://jobs.opensafely.org/>.

### CODE REVIEW

File: analysis/report.ipynb [hunk 1]
```diff
@@ -45,14 +46,12 @@
...
     "\n",
-    "# Introduction\n",
-    "\n",
     "This report displays the schema of the OpenSAFELY-TPP database.\n",
     "It is part of the OpenSAFELY platform's technical documentation\n",
     "and is published at <https://reports.opensafely.org/>.\n",
     "\n",
     "The schema is read from the `OpenSAFELYSchemaInformation` table,\n",
-    "which is refreshed at the same time as the core SystmOne primary care datasets.\n",
+    "which is refreshed at the same time as the core SystemOne primary care datasets.\n",
```
Comment (milanwiedemann) [inline analysis/report.ipynb:54]: I dont think this was a typo, see https://tpp-uk.com/

> "SystmOne, our core product, makes the 50 million electronic health records ... "

Comment (iaindillingham) [reply analysis/report.ipynb:54]: Doh! Thanks.

**Review by ghickman — APPROVED** [review]

File: justfile [hunk 2]
```diff
@@ -108,5 +108,5 @@ fix: devenv
 
 
 # Run JupyterLab
-run: devenv
-    opensafely jupyter
+run *ARGS: devenv
```
Comment (milanwiedemann) [inline justfile:111]: this is more of a question than a comment. I assume you did this so we can specify some of these in the just command?  

```
usage: opensafely jupyter [-h] [--directory DIRECTORY] [--name NAME] [--no-browser] [--port PORT]

options:
  -h, --help            show this help message and exit
  --directory DIRECTORY, -d DIRECTORY
                        Directory to run the jupyter server in (default is current dir)
  --name NAME           Name of docker image (defaults to use directory name)
  --no-browser, -n      Do not attempt to open a browser
  --port PORT, -p PORT  Port to run on
```

Comment (iaindillingham) [reply justfile:111]: Yes, that's correct. Specifically, I wanted to pass `--no-browser` to stop JupyterLab taking control. I often open several environments at the same time; some will start a JupyterLab server. It's distracting if each then opens a new tab in my browser.

**Pull request merged**